packages/web-twig/src/Resources/components/Grid/__tests__/__fixtures__/gridSpacing.twig

```diff
@@ -0,0 +1,53 @@
+<Grid cols="{{ { mobile: 2, tablet: 3, desktop: 4 } }}" spacing="space-1000">
+    <span>col 1</span>
+    <span>col 2</span>
+    <span>col 3</span>
+    <span>col 4</span>
+    <span>col 5</span>
+    <span>col 6</span>
+</Grid>
+
+<Grid cols="{{ { mobile: 2, tablet: 3, desktop: 4 } }}" spacingY="space-400">
+    <span>col 1</span>
+    <span>col 2</span>
+    <span>col 3</span>
+    <span>col 4</span>
+    <span>col 5</span>
+    <span>col 6</span>
+</Grid>
+
+<Grid cols="{{ { mobile: 2, tablet: 3, desktop: 4 } }}" spacingX="space-400">
+    <span>col 1</span>
+    <span>col 2</span>
+    <span>col 3</span>
+    <span>col 4</span>
+    <span>col 5</span>
+    <span>col 6</span>
+</Grid>
+
+<Grid cols="{{ { mobile: 2, tablet: 3, desktop: 4 } }}" spacing="{{ { mobile: 'space-600', tablet: 'space-1000', desktop: 'space-1200' } }}">
+    <span>col 1</span>
+    <span>col 2</span>
+    <span>col 3</span>
+    <span>col 4</span>
+    <span>col 5</span>
+    <span>col 6</span>
+</Grid>
+
+<Grid cols="{{ { mobile: 2, tablet: 3, desktop: 4 } }}" spacingY="{{ { mobile: 'space-600', tablet: 'space-1000', desktop: 'space-1200' } }}">
+    <span>col 1</span>
+    <span>col 2</span>
+    <span>col 3</span>
+    <span>col 4</span>
+    <span>col 5</span>
+    <span>col 6</span>
+</Grid>
+
+<Grid cols="{{ { mobile: 2, tablet: 3, desktop: 4 } }}" spacingX="{{ { mobile: 'space-600', tablet: 'space-1000', desktop: 'space-1200' } }}">
```

Comment:
I would love to see test of only one breakpoint, as in React.

eg. spacingX="{{ { tablet: 'space-1000' } }}"